.github/workflows/ci.yml

```
@@ -30,6 +30,29 @@ jobs:
       - name: "Run tox lint targets for ${{ matrix.python-version }}"
         run: "python -m tox -e check_isort,pep8,packaging"
 
+  build:
+    # This is similar to the job in `release.yml`, but it does NOT upload to PyPI.
+    # We include it here to reduce the likelihood of someone publishing a release
+    # which doesn't build.
+    name: "Build and check package"
```

```suggestion
    name: "Build and check Python package"
```

Not sure "Python package" is the right terminology here, but we should make it explicit we're not talking about eg Debian packaging.